example/huobi/example-huobi-output.beancount

```diff
@@ -1,13 +1,13 @@
 option "title" "测试"
 option "operating_currency" "CNY"
 
-1970-01-01 open Assets:Huobi:Cash
-1970-01-01 open Assets:Huobi:Positions
+1970-01-01 open Assets:FIXME
+1970-01-01 open Assets:Position:FIXME
```

Comment:
Huobi's default accounts should not be FIXME, these default accounts are actually used in the transactions.

Reply:
OK, I will revert that.